backend/shared/src/supabase/utils.ts

```diff
@@ -92,33 +92,76 @@ export async function bulkUpsert<
   await db.none(query)
 }
 
-// Replacement for firebase updateDoc. Updates just the data field (what firebase would've replicated to)
-export async function updateData<T extends TableName>(
+// Replacement for BulkWriter
+export async function bulkUpdateData<T extends TableName>(
```

Comment:
ah I messed up the commit order. it's over here

Reply:
Ah, gotcha. Maybe I was looking at an individual commit by accident?